app/src/main/java/org/mozilla/fenix/library/bookmarks/BookmarkController.kt

```diff
@@ -68,6 +72,9 @@ class DefaultBookmarkController(
 
     private val resources: Resources = activity.resources
 
+    @VisibleForTesting(otherwise = VisibleForTesting.PRIVATE)
+    val maxOpenBeforeWarn = 15
```

Comment:
nit: Could this be marked as internal if it's not intended to be public?
Negligible improvement but it's a flow that we normally use for something that should be private but also want it available in tests.
This could even be a top level const.

Reply:
It was marked as a top level constant during development, but for testing I needed to lower the constant, which is not possible (const), so I switched to a private else in test.
So should I set `internal` and also delete the `Visible for testing` ? Just `internal` ?

Reply:
Just setting this as `internal` and keeping `@VisibleForTesting(otherwise = VisibleForTesting.PRIVATE)` sounds good, thank you!